lib/Generator.js

```diff
@@ -1,88 +1,87 @@
 (function() {
-  'use strict';
+    'use strict';
```

Comment:
I'd rather we didn't have a commit that changes from 2 to 4 spaces - it would make `git blame` less useful. 2 space indent is fine